### PR TITLE
Set thread notifications as seen in mod/display

### LINKED ISSUE #5616: Clear all notifications for a thread after interacting in the thread
Currently, notifications for a thread only are cleared when it is displayed by itself (in `/display`). When a comment or like on a thread in the `/network` page, however, the notifications don't clear, even though it's established I saw there was something new since I interacted with it.

I suggest we clear all notifications for a thread when we like or comment on a thread.

I'm not sure how to proceed, though, I've only seen notifications clear when they are clicked one by one in the popup, but I've seen them disappear by thread on mobile display.

### PR DESCRIPTION
**Pull request by MrPetovan**
Closes #5616

This should remove the need to use "Clear all notifications" as often as it is needed right now.

### CODE REVIEW

**Comment by annando** [comment]
The notifications can be set to either a single notification per thread or to display each single notification, so that you receive multiple notifications for a single thread when (for example) you had been mentioned in different parts of the thread.

Setting the notifications to `read` in this case could be contra productive.

**Comment by MrPetovan** [comment]
Even if notifications are separated, they'll still be there in the notifications menu. This only change the "seen" flag, which isn't a big deal. We aren't deleting anything.

**Comment by annando** [comment]
But maybe you want to work through the unread notifications? I know that @tobiasd is using separate notifications, possibly he could tell his opinion about this?

**Comment by MrPetovan** [comment]
Yes, I'd like him to weigh in on this.

**Comment by MrPetovan** [comment]
Maybe just for grouped notifications?

**Comment by annando** [comment]
This would be a solution.

**Comment by MrPetovan** [comment]
Done.

File: mod/ping.php [hunk 1]
```diff
@@ -289,16 +289,30 @@ function ping_init(App $a)
 		}
 
 		if (DBA::isResult($regs)) {
-			foreach ($regs as $reg) {
+			if (count($regs) <= 1 || DI::pConfig()->get(local_user(), 'system', 'detailed_notif')) {
```
Comment (annando) [inline mod/ping.php:292]: Why do you check here for the detailled notifications?

Comment (MrPetovan) [reply mod/ping.php:292]: We talked about grouping the registration requests, I figured we should leave them as they are now for people who enabled the detailed notifications.

**Comment by tobiasd** [comment]
Best would be, with separate notification, that they are only marked as read if they were actually clicked and thus the comment was highlighted. At the moment they are all marked as read.

**Comment by MrPetovan** [comment]
They're all marked as read for the same thread when you click on any of them? It would make sense with the grouped notification setting, let me see if I can improve this.

**Comment by MrPetovan** [comment]
Done.

**Pull request merged**